### PR TITLE
chore(doc): add test scenarios to verify volume expansion workflow

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

## Pull Request template

Please, go through these steps before you submit a PR.

**What this PR does?**:
This PR adds additional test scenarios to verify volume
expansion feature.

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
NA

**Any additional information for your reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: design/lvm/resize_workflow.md [hunk 1]
```diff
@@ -139,7 +139,9 @@ A test plan will include following test cases:
 - Test volume expansion on statefulset application with multiple replicas.
 - Test volume expansion of thin provisioned volume.
 - Test volume expansion with snapshot(s).
-- Test volume expansion for thick provisioned volume group by increasing size greater then vg size.
+- Test volume expansion of thick provisioned volume group by increasing volume size greater than underlying vg size.
```
Comment (pawanpraka1) [inline design/lvm/resize_workflow.md:142]: let us add what to verify in the test case.

Comment (mittachaitu) [reply design/lvm/resize_workflow.md:142]: Is it something like below?
```
- Test volume expansion with snapshot(s) and verify volume should remain in same size
- Test volume expansion of thick provisioned volume by increasing volume size greater than underlying vg size and verify accessibility of volume
```

Comment (pawanpraka1) [reply design/lvm/resize_workflow.md:142]: yes

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Pull request merged**